youtube_dlc/downloader/external.py

```diff
@@ -202,85 +237,34 @@ class Aria2cFD(ExternalFD):
 
     def _make_cmd(self, tmpfilename, info_dict):
         cmd = [self.exe, '-c']
-        # cmd += self._configuration_args([
-        #    '--min-split-size', '1M', '--max-connection-per-server', '4'])
-        # dn = os.path.dirname(tmpfilename)
-        # if dn:
-        #     cmd += ['--dir', dn]
+        dn = os.path.dirname(tmpfilename)
         if 'url_list' not in info_dict:
             cmd += ['--out', tmpfilename]
+        cmd += self._configuration_args(['-x16', '-j16', '-s16'])
```

Comment:
Let me know your reasoning for removing `'--min-split-size', '1M', '--max-connection-per-server', '4'` and adding these

Reply:
Those options are regarding simultaneous transfers.